### ISSUE TITLE
[Bug]: develop fails to yarn install: @blockaid/ppom@npm:1.1.1: Package not found

### ISSUE THREAD

**Issue by BelfordZ** [MetaMask/metamask-extension#20668]
### Describe the bug

Cannot run current develop branch
```
yarn --immutable
➤ YN0000: · Yarn 4.0.0-rc.48
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 521ms
➤ YN0000: ┌ Post-resolution validation
➤ YN0060: │ @metamask/approval-controller is listed by your project with version 3.4.0, which doesn't satisfy what @metamask/signature-controller requests (^3.5.0).
➤ YN0060: │ @metamask/network-controller is listed by your project with version 12.0.0, which doesn't satisfy what @metamask/assets-controllers requests (^10.3.0).
➤ YN0060: │ @typescript-eslint/eslint-plugin is listed by your project with version 5.30.7, which doesn't satisfy what @metamask/eslint-config-typescript requests (^4.20.0).
➤ YN0060: │ @typescript-eslint/parser is listed by your project with version 5.30.7, which doesn't satisfy what @metamask/eslint-config-typescript requests (^4.20.0).
➤ YN0060: │ eslint is listed by your project with version 8.36.0, which doesn't satisfy what @metamask/eslint-config-jest requests (^7.23.0).
➤ YN0060: │ eslint-plugin-jest is listed by your project with version 26.6.0, which doesn't satisfy what @metamask/eslint-config-jest requests (^24.1.3).
➤ YN0060: │ eslint-plugin-jsdoc is listed by your project with version 39.3.3, which doesn't satisfy what @metamask/eslint-config requests (^36.1.0).
➤ YN0060: │ eslint-plugin-mocha is listed by your project with version 10.1.0, which doesn't satisfy what @metamask/eslint-config-mocha requests (^8.1.0).
➤ YN0060: │ eslint-plugin-prettier is listed by your project with version 4.2.1, which doesn't satisfy what @metamask/eslint-config requests (^3.3.1).
➤ YN0060: │ react-redux is listed by your project with version 7.2.0, which doesn't satisfy what @reduxjs/toolkit requests (^7.2.1).
➤ YN0060: │ typescript is listed by your project with version 4.4.4, which doesn't satisfy what madge requests (^3.9.5 || ^4.9.5 || ^5.0.0).
➤ YN0060: │ webpack is listed by your project with version 5.82.1, which doesn't satisfy what css-loader requests (^4.0.0).
➤ YN0002: │ metamask-crx@workspace:. doesn't provide @testing-library/dom (p3d8b3), requested by @testing-library/user-event
➤ YN0086: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code.
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0035: │ @blockaid/ppom@npm:1.1.1: Package not found
➤ YN0035: │   Response Code: 404 (Not Found)
➤ YN0035: │   Request Method: GET
➤ YN0035: │   Request URL: https://registry.yarnpkg.com/@blockaid/ppom/-/ppom-1.1.1.tgz
➤ YN0000: └ Completed in 0s 801ms

➤ YN0000: · Failed with errors in 1s 424ms
```

### Steps to reproduce

1.clear yarn cache `yarn cache clean --all`
1. yarn install `yarn`

### Error messages or log output

```shell
@blockaid/ppom@npm:1.1.1: Package not found
➤ YN0035: │   Response Code: 404 (Not Found)
➤ YN0035: │   Request Method: GET
➤ YN0035: │   Request URL: https://registry.yarnpkg.com/@blockaid/ppom/-/ppom-1.1.1.tgz
```

### Version

develop

### Build type

None

### Browser

Chrome

### Operating system

Linux

**Comment by subhajit20** [comment]
@BelfordZ Hey is this issue fixed as I am also facing the same problem while running yarn install.

**Comment by Gudahtt** [comment]
Fixed by #20669
